scenarioo-client/test/e2e/pages/branchAliasesPage.ts

```diff
@@ -14,7 +14,7 @@ class BranchAliasesPage {
     }
 
     async assertNumberOfAliases(expectedCount) {
-        const rows = this.branchAliasTable.all(by.css('tbody tr'));
+        const rows = this.branchAliasTable.$$('tbody tr');
```

Comment:
In the "enterAlias" method below we do basically the same, but not with the $$. Is there a specific reason for this? Otherwise it should be aligned with this change.